neptune-bulk-loader/task.ts

```diff
@@ -73,6 +73,12 @@ export const neptuneBulkLoader = async (): Promise<MainResult> => {
 
   await createTermsIndex(esUrl, termsIndex);
 
+  await osClient.deleteByQuery(esUrl, termsIndex, {
+    match: {
+      graph_uri: metadata.namedGraphUri,
```

Comment:
If you change this ES field name you'll need to update it here too.

Reply:
Fixed in 84b7185